src/kemal/config.cr

```diff
@@ -8,162 +10,66 @@ module Kemal
   # Kemal.config
   # ```
   class Config
-    INSTANCE        = Config.new
-    HANDLERS        = [] of HTTP::Handler
-    CUSTOM_HANDLERS = [] of Tuple(Nil | Int32, HTTP::Handler)
-    FILTER_HANDLERS = [] of HTTP::Handler
-    ERROR_HANDLERS  = {} of Int32 => HTTP::Server::Context, Exception -> String
-
-    {% if flag?(:without_openssl) %}
-      @ssl : Bool?
-    {% else %}
-      @ssl : OpenSSL::SSL::Context::Server?
-    {% end %}
-
-    property host_binding, ssl, port, env, public_folder, logging, running
-    property always_rescue, server : HTTP::Server?, extra_options, shutdown_message
-    property serve_static : (Bool | Hash(String, Bool))
+    include ServerConfig
```

Comment:
ServerConfig was extracted because I was planning to have a separate config for the application you want to run and applications you're composing with but didn't follow it through as I didn't implement that functionality.